### PR TITLE
DDP-5277 support revisioning activity naming details

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

Add support for revisioning naming details, so different versions of activity can have different names/titles. We backfill this by creating naming details for each existing version of each existing activity. All the existing study activities should be backfilled this way, so there's no special instructions at deploy time -- just code release.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/src/main/resources/db-changes/schema/DDP-5277-revision-activity-naming-details.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                   i18n_act.second_name,
+                   i18n_act.title,
+                   i18n_act.subtitle,
+                   i18n_act.description
+              from i18n_study_activity as i18n_act
+              join activity_version as act_ver on act_ver.study_activity_id = i18n_act.study_activity_id
+        </sql>
+        <dropTable tableName="i18n_study_activity"/>
+    </changeSet>
+
```
Comment (MocanaAtBroad) [inline pepper-apis/src/main/resources/db-changes/schema/DDP-5277-revision-activity-naming-details.xml:55]: Could we find a way to have these name, subtitle be instance of Template? Worries me that we have another table with translation text.

Comment (yufengwng) [reply pepper-apis/src/main/resources/db-changes/schema/DDP-5277-revision-activity-naming-details.xml:55]: We have other "translation text" stuff as well, such as "summaries". I think moving to Templates is a much bigger change. Can we tackle that separately?

Comment (MocanaAtBroad) [reply pepper-apis/src/main/resources/db-changes/schema/DDP-5277-revision-activity-naming-details.xml:55]: OK. I see just migrating existing stuff. Never mind.

**Comment by yufengwng** [comment]
No rush on this PR since we don't need this yet for TestBoston.

**Review by MocanaAtBroad — APPROVED** [review]

**Review by MocanaAtBroad — DISMISSED** [review]

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**